### PR TITLE
fixed side panel minimize button overlapping with text bug

### LINKED ISSUE #6629: Virtual Event Side Panel: Minimize Button overlaps text
The minimize side panel button overlaps the date and speakers button. Please ensure the right side of the date does not overlap.

Screenshot from 2021-02-07 12-14-44

### PR DESCRIPTION
**Pull request by astromanish**
Fixes #6629 

#### Short description of what this resolves:

Now minimize button wouldn't overlap with date, time text, or with the speaker/schedule button as shown below.

Screenshot (160)
Screenshot (161)

#### Changes proposed in this pull request:

- side panel date, time wrapped into a div, and right margin added to it
- top, bottom padding added to the schedule and speaker button enclosing div

#### Checklist

- [✓] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [✓] My branch is up-to-date with the Upstream `development` branch.
- [✓] The acceptance, integration, unit tests and linter pass locally with my changes 
- [✓] I have added tests that prove my fix is effective or that my feature works
- [✓] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by astromanish** [comment]
@mariobehling Done! Sir

**Comment by afzal442** [comment]
Still not fixed. Look at the below section as Some checks were not successful.

File: app/styles/components/stream.scss [hunk 1]
```diff
@@ -57,3 +57,7 @@
   overflow: hidden;
   z-index: 1;
 }
+
+.date-side-panel {
+  margin-right: 1.5rem;
+}
```
Comment (maze-runnar) [inline app/styles/components/stream.scss:63]: there are already classes named `mr-2 mr-4 ..` use them

File: app/components/public/stream/side-panel.hbs [hunk 2]
```diff
@@ -12,9 +12,12 @@
         <p class="text-white">{{@event.name}}</p>
       </a>
     </div>
-    <Public::EventDateTime @event={{@event}} />
+    <div class="date-side-panel">
+      <Public::EventDateTime @event={{@event}} />
+    </div>
```
Comment (maze-runnar) [inline app/components/public/stream/side-panel.hbs:17]: add mr-4 see i it's enough.

**Comment by maze-runnar** [comment]
fixed here: https://github.com/fossasia/open-event-frontend/pull/6890

**Pull request closed without merging**